src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -418,9 +418,15 @@ Rustdoc only supports Rust source code and Markdown input formats. If the
 file ends in `.md` or `.markdown`, `rustdoc` treats it as a Markdown file.
 Otherwise, it assumes that the input file is Rust.
 
+# Unstable command line arguments
+
 ## `--nocapture`
 
 When this flag is used with `--test`, the output (stdout and stderr) of your tests won't be
 captured by rustdoc. Instead, the output will be directed to your terminal,
 as if you had run the test executable manually. This is especially useful
 for debugging your tests!
+
+## `--check`
+
+Runs rustdoc checks without documenting nor testing anything.
```

Comment:
This seems weirdly terse compared to the rest of this docs page. Sentence fragments make sense in `--help` output, since you're trying to fit everything in one column, but in the long-form owner's manual, we should probably use full sentences and examples.

I hate to bikeshed, but here's what I'd write instead:

> When this flag is supplied, rustdoc will type check and lint your code, but will not generate any documentation or run your doctests.
>
> Using this flag looks like:
>
>     rustdoc -Z unstable-options --check src/lib.rs
>

Reply:
Bikeshedding is fine, don't worry. I'm not very good at this so any help is very appreciated!